### DISCUSSION TITLE
Outline site no longer available

### DISCUSSION THREAD

**Discussion by strawberryfly** [outline/outline#4750] [Self Hosting]
After upgrading the docker from 0.67pre release I can no longer access outline. Unfortunately no error message is displayed. As soon as I go to version 0.67.1 or 0.67.2, I can't get to the page anymore. I don't use Redis username and I don't have a proxy in front of it. After a downgrade Outline is reachable again. 

Does anyone have an idea why this could be?

**Comment by tommoor** [comment]
You should post your server logs, it won't be possible for anyone to help you without that.

**Comment by strawberryfly** [comment]
attachted the logs. It looks like the port 3000 will disappear in he URL, when i try to connect. I cleaned my Browser and installed a new one

`
yarn run v1.22.18
$ node ./build/server/index.js
{"label":"lifecycle","level":"info","message":"\nIs your team enjoying Outline? Consider supporting future development by sponsoring the project:\n\nhttps://github.com/sponsors/outline\n"}
{"label":"lifecycle","level":"info","message":"Starting collaboration service"}
{"label":"lifecycle","level":"info","message":"Starting websockets service"}
{"label":"lifecycle","level":"info","message":"Starting worker service"}
{"label":"lifecycle","level":"info","message":"Starting web service"}
{"label":"lifecycle","level":"info","message":"Listening on https://localhost:3000"}

`

**Comment by strawberryfly** [comment]
.env

`
DATABASE_URL=postgres://user:XYP@localhost:5432/outline
REDIS_URL=redis://localhost:6379
URL=http://outline.company.xy:3000
PORT=3000
FORCE_HTTPS=True
OIDC_CLIENT_ID=wikiclient
OIDC_USERNAME_CLAIM=preferred_username
OIDC_CLIENT_SECRET=xy
OIDC_AUTH_URI=https://keycloak.company.xy:8443/realms/wiki/protocol/openid-connect/auth
OIDC_TOKEN_URI=https://keycloak.company.xy:8443/realms/wiki/protocol/openid-connect/token
OIDC_USERINFO_URI=https://keycloak.company.xy:8443/realms/wiki/protocol/openid-connect/userinfo

`

**Comment by strawberryfly** [comment]
ok, after a bit of trial and error i found out that it is because of the HTTP_Force=True paramter. If it is set to false, outline can be called again. Strange because i don't have a proxy in between. However, now my login always fails. I will try a little more and let you know what the problem is :) 

` GET /auth/oidc.callback?state=3a33c597c73c80de&session_state=e78ff381-eaf3-4da4-8b91-d598076c8a84&code=f2cc91aa-d06a-4a39-8626-30c1dec51409.e78ff381-eaf3-4da4-8b91-d598076c8a84.7dab3518-447e-4fc7-a399-1520ba54b70d"}
{"error":"Failed to obtain access token","stack":"InternalOAuthError: Failed to obtain access token\n    at OAuth2Strategy._createOAuthError (/opt/outline/node_modules/passport-oauth2/lib/strategy.js:423:17)\n    at /opt/outline/node_modules/passport-oauth2/lib/strategy.js:177:45\n    at /opt/outline/node_modules/oauth/lib/oauth2.js:191:18\n    at ClientRequest.<anonymous> (/opt/outline/node_modules/oauth/lib/oauth2.js:162:5)\n    at ClientRequest.emit (node:events:526:28)\n    at ClientRequest.emit (node:domain:475:12)\n    at TLSSocket.socketErrorListener (node:_http_client:442:9)\n    at TLSSocket.emit (node:events:526:28)\n    at TLSSocket.emit (node:domain:475:12)\n    at emitErrorNT (node:internal/streams/destroy:157:8)\n    at emitErrorCloseNT (node:internal/streams/destroy:122:3)\n    at processTicksAndRejections (node:internal/process/task_queues:83:21)","level":"error","message":"Error during authentication"}
`
